### ISSUE TITLE
minikube:  upload data  :  Connection refused, Please check if the fate flow service is started

### ISSUE THREAD

**Issue by ghl116** [FederatedAI/KubeFATE#602]
**What deployment mode you are use?  **
minikube 

**What KubeFATE and FATE version you are using?   fate  1.7.2  KubeFATE: 1.4.3

**MUST** Please state the KubeFATE and FATE version you found the issue     

**What OS you are using for docker-compse or Kubernetes? Please also clear the version of OS.  **

 - OS: [e.g. iOS] centos
 - Version [e.g. 22] 7.9

**To Reproduce**
Follow instructions in [Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube](https://github.com/FederatedAI/KubeFATE/blob/v1.7.2/docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube.md) 

Clear how to reproduce your problem.

**What happen?**

after installation, 
1、upload data error with jupyter book.  
```
ValueError: job submit failed, err msg: {'retcode': 100, 'retmsg': 'Connection refused, Please check if the fate flow service is started'}
```
2、upload  data error with  flow 
```
flow data upload -c examples/dsl/v2/upload/upload_conf.json
```

**Screenshots**

![image](https://user-images.githubusercontent.com/19288752/165665853-208bd196-591d-44ec-91aa-cfeb322960f9.png)

**Additional context**

**Comment by JingChen23** [comment]
https://github.com/FederatedAI/KubeFATE/blob/v1.8.0/docker-build/client/pipeline/pipeline_tutorial_upload.ipynb
Please try this script from 1.8.0. The script you are showing from 1.7.2 has a known issue. It needs to be modified a little bit to work. We have done this in 1.8.0

**Comment by JingChen23** [comment]
We will support 2 backward versions, 1.8.0 is one of them and doesn't have this issue, the other one will be 1.9.0 which will be released in the next half of the year. Thus, for this issue reported for 1.7.2, close it for now.
If a patch release of 1.7.2 is coming earlier than 1.9.0, then we will use this issue to track the fix on the patch release of 1.7.2.
